h2/src/test/org/h2/test/scripts/ddl/createTable.sql

```diff
@@ -131,3 +131,9 @@ SELECT COLUMN_NAME, COLUMN_TYPE FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME
 
 DROP TABLE TEST;
 > ok
+
+CREATE TABLE IF NOT EXISTS TEST(ID BIGINT(20) NOT NULL AUTO_INCREMENT PRIMARY KEY COMMENT '字典ID', FIELD_NAME VARCHAR(100) NOT NULL COMMENT '字段名称', TABLE_NAME VARCHAR(100) NOT NULL COMMENT '表名');
```

Comment:
Please, do not use non-ASCII characters in test cases, they aren't needed here.

Also it's reasonable to test that comment was really set with `SCRIPT TABLE TEST` command or in some other way.